cg_openmm/utilities/iotools.py

```diff
@@ -146,7 +146,7 @@ def write_pdbfile_without_topology(CGModel, filename, energy=None):
             bead_index = bead_index + 1
```

Comment:
We could update the pdb writer to use the monomer name instead of 'M' for all cases as the residue name. Should I make it a separate issue?

Reply:
Ah, good idea.  I just added that in, let me know what you think.